CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- [BUGFIX] fix!: swizzle URLSessionTask for network instrumentation. See [#1394][]
```

Comment:
Probably better to describe what do we fix instead of how

Reply:
Yes, looks like I have to write a whole article now on this.